daft/filesystem.py

```diff
@@ -214,58 +196,89 @@ def _resolve_path_and_filesystem(
 
     Args:
         path: A single file/directory path.
-        filesystem: The filesystem implementation that should be used for
-            reading these files. If None, a filesystem will be inferred. If not
-            None, the provided filesystem will still be validated against the
-            filesystem inferred from the provided path to ensure compatibility.
+        io_config: A Daft IOConfig that should be best-effort applied onto the returned
+            FileSystem
+        cached_filesystem: If provided, this filesystem will be validated against the
```

Comment:
I dont see this passed in

Reply:
Removed - bad docstring from a previous iteration of the branch